src/pages/mijn-bureau-erkenningen/inloggevens-kwijt.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Inloggevens kwijt
 ---
-Bent u uw inloggevens kwijt? 
+
+Bent u uw inloggevens kwijt?
 
```

Comment:
This page is not reachable from the menu. Also, check the spelling. Global search for inloggevens > inloggegevens. So rename the file.